prow/config.yaml

```diff
@@ -152,28 +152,45 @@ presubmits:
     trigger: "(?m)^/test( all| pull-cadvisor-e2e),?(\\s+|$)"
     labels:
       preset: service-account
-      ssh-preset: k8s-ssh
```

Comment:
keep the preset and remove the boilerplate :-)

Reply:
is this what you meant?

Reply:
keep the git-cache part though :-)